h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -427,6 +438,15 @@ public BinomialModelPrediction predictBinomial(RowData data, double offset) thro
     return p;
   }
 
+  private String[] leafNodeAssignment(RowData data) throws PredictException {
```

Comment:
I would even think about making this method public for the users who want finer grade control over the API.